src/Settings/components/larivaar.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { useState } from "react";
+import { Text } from "react-native";
+import { ListItem, BottomSheet, Avatar, Icon, Switch } from "@rneui/themed";
+import STRINGS from "../../common/localization";
+import { toggleLarivaar, toggleLarivaarAssist } from "../../common/actions";
+import { useSelector } from "react-redux";
+import colors from "../../common/colors";
+import styles from "../styles";
+
+function LarivaarComponent({ isNightMode, dispatch }) {
+  const { isLarivaar, isLarivaarAssist } = useSelector((state) => state);
+
+  const LarivaarAssist = () => (
```

Comment:
As LarivaarAssist is a function and not a component, you can use camelcase instead - larivaarAssist.

Reply:
fixed